lib/livebook/storage/ets.ex

```diff
@@ -73,9 +76,17 @@ defmodule Livebook.Storage.Ets do
 
   @impl GenServer
   def init(_opts) do
-    table = :ets.new(@table_name, [:named_table, :protected, :duplicate_bag])
+    # Make sure that this process does not terminate abruptly
+    # in case it is persisting to disk. terminate/2 is still a no-op.
+    Process.flag(:trap_exit, true)
+
+    file_path = config_file_path()
 
-    {:ok, %{table: table}}
+    # enable passing table name for testing purposes
+    table = load_or_create_table(file_path)
+    :persistent_term.put(__MODULE__, table)
+
+    {:ok, %{table: table, file_path: file_path}}
```

Comment:
Now that we don't pass `file_path` in options, we can just call `config_file_path()` whenever needed, no?